x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/transform_health_colored_dot.tsx

```diff
@@ -18,14 +18,22 @@ import {
 
 interface TransformHealthProps {
   healthStatus: TransformHealth;
+  compact?: boolean;
 }
 
-export const TransformHealthColoredDot: FC<TransformHealthProps> = ({ healthStatus }) => {
-  return (
+export const TransformHealthColoredDot: FC<TransformHealthProps> = ({
+  healthStatus,
+  compact = true,
+}) => {
+  return compact ? (
```

Comment:
super duper nit: Suggestion to have constants for these for readability.

```
const transformHealthDescription = TRANSFORM_HEALTH_DESCRIPTION[healthStatus]
const transformHealthColor = TRANSFORM_HEALTH_COLOR[healthStatus]
const transformHealthLabel = TRANSFORM_HEALTH_LABEL[healthStatus]

```

Reply:
Now fixed in follow up: https://github.com/elastic/kibana/pull/151415